sdk/storage/azure-storage-queues/CMakeLists.txt

```
@@ -0,0 +1,87 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.13)
+project(azure-storage-queues LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../../cmake-modules")
+
+include(AzureVcpkg)
+include(AzureVersion)
+include(AzureCodeCoverage)
+include(AzureTransportAdapters)
+include(AzureDoxygen)
+include(AzureGlobalCompileOptions)
+
+az_vcpkg_integrate()
+
+if(NOT AZ_ALL_LIBRARIES)
+  find_package(azure-storage-common-cpp CONFIG QUIET)
+  if(NOT azure-storage-common-cpp_FOUND)
+    find_package(azure-storage-common-cpp REQUIRED)
```

```suggestion
  find_package(azure-storage-common-cpp "12.0.0" CONFIG QUIET)
  if(NOT azure-storage-common-cpp_FOUND)
    find_package(azure-storage-common-cpp "12.0.0" REQUIRED)
```
(this was added just a day ago, and this is how we should do it now. If Queues require updates to Storage-Common and would require greater version, update it here)